docs/Tailwind/GettingStarted.stories.mdx

```diff
@@ -0,0 +1,120 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { CATEGORIES } from "../../storybook/constants"
+import TailwindPlay from "./assets/tailwind-play.gif"
+import CssGeneration from "./assets/css-generation.png"
+import InlineFold from "./assets/inline-fold.gif"
+
+<Meta title={`${CATEGORIES.tailwind}/Getting Started`} />
+
+# Getting Started  
+
+This page describes the steps needed to use our [Kaizen Tailwind preset](https://github.com/cultureamp/kaizen-design-system/tree/main/packages/tailwind), not Tailwind itself.  
+To learn more about Tailwind and how to install it in your project, check out there docs [here](https://tailwindcss.com/docs/installation).
+
+- [Install the preset](#install-the-preset)
+- [Implement the preset](#implement-the-preset)
+- [Add the Tailwind directives](#add-the-tailwind-directives)
+- [Useful tools](#useful-tools)
+
+## Install the preset
```

Comment:
I think you should have 2 sub sections here, one for "automatic install" describing that FE template already has this and then "manual install"

Reply:
Yep good idea - Given that the FE template should be good to go out of the box, I've just numbered the steps and instructed FE-template users to skip the steps that aren't needed. More concise this way.